demo/dlb-init.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+ENABLED_DLB_PF_PCIIDS=${ENABLED_DLB_PF_PCIIDS:-270b 2710 2714}
+NUMVFS=${NUMVFS:-4}
+
+bind() {
+    BSF=$1
+    DRIVER=$2
+
+    LOOP=0
+    while [ "$LOOP" -lt  5 ]
+    do
+        if echo -n 2> /dev/null "$BSF" > /sys/bus/pci/drivers/"$DRIVER"/bind; then
+            break
+        else
+           LOOP=$(("$LOOP" + 1))
+           echo -n 2> /dev/null "$BSF" > /sys/bus/pci/devices/"$BSF"/driver/unbind
+            sleep 1
+        fi
+    done
+}
+
+unbind() {
+    BSF=$1
+    OLD_DRIVER=$2
+
+    LOOP=0
+    while [ "$LOOP" -lt  5 ]
+    do
+        if echo -n 2> /dev/null "$BSF" > /sys/bus/pci/drivers/"$OLD_DRIVER"/unbind;
+        then
+            break
+        else
+           LOOP=$(("$LOOP" + 1))
+            sleep 1
+        fi
+    done
+}
+
+#get dlb pf pci ids
+DEVS=$(for pf in $ENABLED_DLB_PF_PCIIDS; do lspci -n | grep -e "$pf" | grep -o -e "^\\S*"; done)
+for dev in $DEVS; do
+  DEVPATH="/sys/bus/pci/devices/0000:$dev"
+
+  #enable sriov_numvfs
+  unbind 0000:"$dev" dlb2
+  bind 0000:"$dev" dlb2
```

Comment:
any reason for this?

Reply:
Yes!
It's because of the following command.
`echo "$NUMVFS" | tee -a "$DEVPATH"/sriov_numvfs`

If we do not 'reset' the number, and in case sriov_numvfs is already set, then setting the value with another number fails.
So, I did it.

Reply:
With QAT we skip writing if someone has already configured the device:
https://github.com/intel/intel-device-plugins-for-kubernetes/blob/685ed6ecece7c8c365e568f6a4ad776399af5a0d/demo/qat-init.sh#L8

Reply:
Ok, that would be a better solution!

Reply:
@mythi But,,, actually I found one problem.
If sriov_numvfs is set, but "num_atomic_inflights", "num_dir_credits", or such files are not configured, then it still skips.

Reply:
As far as I remember I had to first unbind then set all resource values and only after that bind. Setting values for already bind devices didn't work for me. At least in Simics.

Reply:
> If sriov_numvfs is set, but "num_atomic_inflights", "num_dir_credits", or such files are not configured, then it still skips.

I think that's fine. They may have been configured for other usages (e.g., `vfio-pci`) so we better not to touch them at all.

Reply:
@bart0sh that must be something for setting values of 'vfs'. This line is before then. So, I thnk we do not need unbind and bind for this. For setting values of 'num_something', then, we unbind before setting.